bool has_motion
bool is_collider
bool is_tracked
bool is_stopmark
bool is_lanemark
bool is_ignored
bool is_hero
